CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 - Allow mmap to use new JDK-19 preview APIs in Apache Lucene 9.4+ ([#5151](https://github.com/opensearch-project/OpenSearch/pull/5151))
 - Add events correlation engine plugin ([#6854](https://github.com/opensearch-project/OpenSearch/issues/6854))
 - Add connectToNodeAsExtension in TransportService ([#6866](https://github.com/opensearch-project/OpenSearch/pull/6866))
+- Adding task cancellation timestamp in task API ([#7455](https://github.com/opensearch-project/OpenSearch/pull/7455))
```

Comment:
Please add this entry to the `Unreleased 2.x` section if you plan on backporting it to `2.x`

Reply:
Added.

Reply:
There should only be one entry in the changelog, and this one should be in the Unreleased 2.x section. See [this doc](https://github.com/opensearch-project/OpenSearch/blob/90fefa262c7089eefe846e49c3acb96469d98eb6/CONTRIBUTING.md?plain=1#L147-L154) for details.

Also, please use the imperative mood to match the other entries ("Add" versus "Added").